server/api/batch_inventory.py

```diff
@@ -518,7 +518,11 @@ def process_ess_md():
         cvr_file.close()
 
         cvr_file_readers = [
-            csv_reader_for_cvr(retrieve_file(join(working_directory, file_name)))
+            csv_reader_for_cvr(
+                # TODO: Revisit file cleanup holistically
```

Comment:
Even when we're using `retrieve_file` we aren't calling `close` on files as far as I can tell so no regression here as far as I can tell. Python garbage collection should eventually take care of things but might be worth revisiting file cleanup holistically.